packages/firebase_auth/firebase_auth/example/lib/main.dart

```diff
@@ -31,7 +31,8 @@ Future<void> main() async {
     );
   }
 
-  await FirebaseAuth.instance.useAuthEmulator('localhost', 9099);
+  /* Uncomment the line below if you are running the example from a emulator, otherwise this line will cause a connection error */
```

Comment:
Would be nice to avoid bloc comments: https://dart.dev/guides/language/effective-dart/documentation#dont-use-block-comments-for-documentation

Reply:
Hey @seanwestfall, thanks for the PR! Please remove block comment and follow the same implementation as [firestore example](https://github.com/firebase/flutterfire/blob/master/packages/cloud_firestore/cloud_firestore/example/lib/main.dart#L10-L12) regarding setting/unsetting emulator 👍